content/api/commands/origin.md

```diff
@@ -0,0 +1,512 @@
+---
+title: origin
+---
+
+Visit multiple domains of different
+[origin](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy#definition_of_an_origin)
+in a single test.
+
+In normal use, a single Cypress test may only run commands in a single origin, a
+limitation determined by standard web security features of the browser. The
+`cy.origin()` command allows your tests to bypass this limitation.
+
+<Alert type="warning">
+
+<strong class="alert-header"><Icon name="exclamation-triangle"></Icon>
+Experimental</strong>
+
+The `origin` API is currently experimental, and can be enabled by setting
+the [`experimentalSessionAndOrigin`](/guides/references/experiments) flag
+to `true` in the Cypress config.
+
+Enabling this flag does the following:
+
+- It adds the [`cy.session()`](/api/commands/session) and `cy.origin()`
+  commands, and [`Cypress.session`](/api/cypress-api/session) API.
+- It adds the following new behaviors (that will be the default in a future
+  major update of Cypress) at the beginning of each test:
+  - The page is cleared (by setting it to `about:blank`).
+  - All active session data (cookies, `localStorage` and `sessionStorage`)
+    across all domains are cleared.
+- It overrides
+  the [`Cypress.Cookies.preserveOnce()`](/api/cypress-api/cookies#Preserve-Once) and
+  [`Cypress.Cookies.defaults()`](/api/cypress-api/cookies#Defaults) methods.
+- Cross-domain requests will no longer fail immediately, but instead, time out
+  based on [`pageLoadTimeout`](/guides/references/configuration#Timeouts).
+- Tests will no longer wait on page loads before moving on to the next test.
+
+Because the page is cleared before each
+test, [`cy.visit()`](/api/commands/visit) must be explicitly called in each test
+to visit a page in your application.
+
+</Alert>
+
+## Syntax
+
+```js
+cy.origin(url, callbackFn)
+cy.origin(url, options, callbackFn)
+```
+
+### Usage
+
+**<Icon name="check-circle" color="green"></Icon> Correct Usage**
+
+```js
+const hits = getHits() // Defined elsewhere
+// Run commands in secondary origin, passing in serializable values
+cy.origin('https://www.acme.com', { args: { hits } }, ({ hits }) => {
+  // Inside callback baseUrl is https://www.acme.com
+  cy.visit('/history/founder')
+  // Commands are executed in secondary origin
+  cy.get('h1').contains('About our Founder, Marvin Acme')
+  // Passed in values are accessed via callback args
+  cy.get('#hitcounter').contains(hits)
+})
+// Even though we're outside the secondary origin block,
+// we're still on acme.com so return to baseUrl
+cy.visit('/')
+// Continue running commands on primary origin
+cy.get('h1').contains('My cool site under test')
+```
+
+**<Icon name="exclamation-triangle" color="red"></Icon> Incorrect Usage**
+
+```js
```

Comment:
IMO it might be easier to break this into two examples since both commands within the callback are incorrect but technically the second command wold never execute